publisher-sdk/src/main/java/com/criteo/publisher/BidManager.java

```diff
@@ -217,12 +225,15 @@ public void onApplicationStopped() {
    * @param adUnits list of ad units to prefetch
    */
   public void prefetch(@NonNull List<AdUnit> adUnits) {
-    List<List<CacheAdUnit>> requestedAdUnitsChunks = adUnitMapper.mapToChunks(adUnits);
+    if (bidPrefetchRateLimiter.canPrefetch()) {
+      bidPrefetchRateLimiter.setPrefetchTime();
```

Comment:
Nit: `set` prefix is usually used for setter method and it may be weird to see a `set*` method without argument.
What do you think about renaming it to show that the rate limiter handle this itself ? Like `markPrefetchTime` ?